custom-commands/quick-start/DeviceState/DeviceState/.gitignore

```diff
@@ -0,0 +1,264 @@
+## Ignore Visual Studio temporary files, build results, and

```

Comment:
Do you need this gitignore? I think there's already one in the root folder.

Reply:
Good catch! Removed